src/content/docs/en/technology/chain/rollup.mdx

```diff
@@ -128,8 +128,9 @@ This section describes the codec of three data structures in the Rollup contract
 | `l1MessagePopped`        | 8       | `uint64`    | 9      | The number of L1 messages poped in the batch                    |
 | `totalL1MessagePopped`   | 8       | `uint64`    | 17     | The number of total L1 messages popped after the batch          |
 | `dataHash`               | 32      | `bytes32`   | 25     | The data hash of the batch                                      |
-| `parentBatchHash`        | 32      | `bytes32`   | 57     | The parent batch hash                                           |
-| `skippedL1MessageBitmap` | dynamic | `uint256[]` | 89     | A bitmap to indicate which L1 messages are skipped in the batch |
+| `blobVersionedHash`      | 32      | `bytes32`   | 57     | The versioned hash of the blob with this batch’s data           |
```

Comment:
We can't just use a footnote here. We need this published for people upgrading, but also should document what the "mainnet" version is. Is this a usecase for our "tab" UI element? Scroll as default Scroll Sepolia as 2nd choice?

Reply:
I've implemented the tabs but the styling of the component feels unpolished, I don't think it's obvious what is being toggled by the tabs

@zzq0826 could we update the Tabs component?

Reply:
Marking Resolved, tabs will be another PR